MinecraftClient/McClient.cs

```diff
@@ -75,7 +75,14 @@ public class McClient : IMinecraftComHandler
         // server TPS
         private long lastAge = 0;
         private DateTime lastTime;
-        private Double serverTPS = 0;
+        private double serverTPS = 0;
+        private double averageTPS = 20;
+        private const int maxSamples = 5;
+        private List<double> tpsSamples = new List<double>(maxSamples);
+        private double sampleSum = 0;
+
+        // players latency
+        private Dictionary<string, int> playersLatency = new Dictionary<string, int>();

```

Comment:
I see this being populated but not used. What is its purpose?

Reply:
I planned to make a command for viewing other player's ping. Also tried include player latency as tps calculate factor but seems like it doesn't affect much so yes, it is not used now.

Reply:
Perhaps my Chat Bot which I made a long time ago will help in creating this.
https://pastebin.com/ZCssAJih